server/src/main/java/org/elasticsearch/Build.java

```diff
@@ -115,12 +116,22 @@ private static Build findLocalBuild() {
 
         final String flavor = "default";
         String minWireCompat = Version.CURRENT.minimumCompatibilityVersion().toString();
-        String minIndexCompat = Version.CURRENT.minimumIndexCompatibilityVersion().toString();
+        String minIndexCompat = minimumCompatString(IndexVersion.MINIMUM_COMPATIBLE);
         String displayString = defaultDisplayString(type, hash, date, version);
 
         return new Build(flavor, type, hash, date, isSnapshot, version, minWireCompat, minIndexCompat, displayString);
     }
 
+    public static String minimumCompatString(IndexVersion minimumCompatible) {
+        if (minimumCompatible.before(IndexVersion.V_8_11_0)) {
```

Comment:
Shouldn't this be 8.10?

Reply:
Its `before`, not `onOrBefore`, so its exclusive of 8.11